geo/src/algorithm/geodesic_destination.rs

```diff
@@ -50,7 +50,11 @@ mod test {
     fn returns_a_new_point() {
         let p_1 = Point::new(9.177789688110352, 48.776781529534965);
         let p_2 = p_1.geodesic_destination(45., 10000.);
-        assert_eq!(p_2, Point::new(9.27411867078536, 48.8403266058781));
+
+        let expected = Point::new(9.27411867078536, 48.8403266058781);
+        assert_relative_eq!(p_2.x(), expected.x(), epsilon = 1.0e-6);
```

Comment:
Since we've implemented some approx traits for Point, you should be able to just `assert_relative_eq(p2, expected, epsilon = 1.e-6)`.

Reply:
Oh I didn't know that!  Good catch. Fixed